index.html

```diff
@@ -151,47 +151,47 @@
         Introduction
       </h2>
       <p>
+        The Push API allows a <a>webapp</a> to communicate with a <a>user agent</a> asynchronously.
+        This allows a <a>webapp</a> to provide the <a>user agent</a> with time-sensitive
+        information whenever that information becomes known, rather than waiting for a user to open
+        the <a>webapp</a>.
+      </p>
+      <p>
         As defined here, <a>push services</a> support delivery of <a>application server</a>
-        messages in the following contexts and related use cases:
+        messages at any time.
+      </p>
+      <p>
+        In particular, a <a>push message</a> will be delivered to the <a>webapp</a> even if that
+        <a>webapp</a> is not currently active in a browser window: this relates to use cases in
+        which the user may close the <a>webapp</a>, but still benefits from the <a>webapp</a> being
+        able to be restarted when a <a>push message</a> is received. For example, a <a>push
+        message</a> might be used to inform the user of an incoming WebRTC call.
+      </p>
+      <p>
+        A <a>push message</a> can also be sent when the <a>user agent</a> is temporarily offline.
+        In support of this, the <a>push service</a> stores messages for the <a>user agent</a> until
+        the <a>user agent</a> becomes available. This supports use cases where a <a>webapp</a>
+        learns of changes that occur while a user is offline and ensures that the <a>user agent</a>
+        can be provided with relevant information in a timely fashion. For example, a user that is
+        away from network access might receive a new message; the message is stored by the <a>push
+        service</a> until the <a>user agent</a> becomes reachable and the message can be delivered.
```

Comment:
"For example, a user that...can be delivered." just repeats the previous sentence. I think we can drop it.